update/Makefile

```diff
@@ -68,7 +70,7 @@ env-down:                       ## Stop development environment
 	docker-compose down --volumes --remove-orphans
 
 install-dev-tools:
-	docker exec pmm-update-server /root/go/src/github.com/percona/pmm/update/.devcontainer/install-dev-tools.sh
+	.devcontainer/install-dev-tools.sh
```

Comment:
This target should be moved to `Make.devcontainer` then

Reply:
Not sure I understand. Can you suggest a fix?

Reply:
Would that be a proper solution? https://github.com/percona/pmm/pull/2264/commits/fff4312b409b82ce7d317ac94ff1a8b7159e95e0

Reply:
`Makefile` used on the host machine, if user will execute it then `install-dev-tools.sh` will try to install tools on the host machine, but it should happen only inside devcontainer.

Reply:
Well, I doubt that things like `check`, `test`, `test-race` will run outside of the container. There is a good mix of targets that will and won't. 

Anyway, this seems to be blocking the review so I'm revering it for now.